specs/latest/2.0/index.html

```diff
@@ -3320,6 +3304,22 @@ <h3><a name="COPYING_BUFFERS">Copying Buffers</a></h3>
         Same as with <a href="#BUFFER_OBJECT_BINDING">Buffer Object Binding</a> restrictions above.
     </div>
 
+    <h3>Preventing undefined behavior with Transform Feedback</h3>
+
+    <p>
+        A buffer which is simultaneously bound to a TRANSFORM_FEEDBACK_BUFFER binding point and any other
```

Comment:
The example seems to say this, but maybe clarify: The TRANSFORM_FEEDBACK_BUFFER binding point of the currently bound TF object only, right, not any other unbound ones??

Reply:
Yes, I'll clarify that.